_episodes/64.md

```diff
@@ -0,0 +1,81 @@
+
+* [Sebastian Bernauer](https://www.linkedin.com/in/sebastian-bernauer-622b95167),     
+  Software Developer at [Stackable]({{site.url}}/users#stackable)
+* [Sönke Liebau](https://www.linkedin.com/in/soenkeliebau/), Co-Founder and CPO
+  at [Stackable]({{site.url}}/users#stackable)
+
+## Releases and news
+
+[Trino 454](https://trino.io/docs/current/release/release-454.html)
+
+As usual, numerous performance improvements, bug fixes, and other features
+have been added as well.
+
+Other noteworthy topics:
+
+none yet
+
+
+
+## OPA
+
+join us to
+talk about their experience with using
+<a href="{{site.url}}/ecosystem/add-on.html#open-policy-agent">Open Policy Agent</a>
+for access control with Trino.</dd>
+
+I personally could think of the following things to discuss:
+* Mention that we are building on top of https://www.youtube.com/watch?v=fbqqapQbAv0, users should watch that for motivation and technical details of the OPA plugin
+* Brief presentation of Stackable
+* Talk about recent development of the OPA plugin, e.g. https://github.com/trinodb/trino/pull/21997 which batches column masking calls (so one call per table instead of per columns)
+* Mention our compatibility layer, where we wrote OPA rego rules which can take in the classic file based access control JSON and try to mimic Trinos behavior. This allows for easier migration file based -> OPA and could serve as a starting point. They are Open-Source at https://github.com/stackabletech/trino-operator/tree/main/tests/templates/kuttl/opa-authorization/trino_rules (sorry for OSL3, we can talk about licensing them separately ;)). Mention our customer had performance problems due to the way the rules work (they are imperative top to bottom, first match wins) and decided to write their own rego rules
+* Quickly walk throw our demo showcasing row-level and column-level security. But I would not go too crazy here and would link to https://www.youtube.com/watch?v=ATlq_l3WNiA for details. This demo uses the compatibility layer mentioned above
+* Ideally get our customer to present their custom rego rules - this would showcase the point of maximum flexibility in authorization.
+* Talk about measures we took to "product-ionize" Trino on Kubernetes: Graceful shutdown, PodDisruptionBudgets, anti-affinity
+* Talk about trino-lb, potentially showcase a setup where we autoscale the number of Trino clusters running. Clarify the differences between trino-gateway and trino-lb. I hope to get some Grafana dashboard screenshots from our customer.
+* Future planned steps (e.g. Autoscaling of Trino workers out of the box based on metrics - so not only CPU or memory)
+I know this is more than we originally planned in https://trino.io/broadcast/. But I feel we already have a somewhat detailed talk on Trino -> OPA and there could be some lessons learned in particular for Kubernetes. As we planned a community broadcast on trino-lb anyway.
+What do you think of the extended scope?
+Happy about any feedback!
```

Comment:
That's the current agenda we have in our slides
```suggestion
The agenda contains the following points:

* What is Stackable?
* Open Policy Agent (OPA) authorization plugin
  * History
  * Recent development
  * Compatibility layer to Trino’s File-based access control
  * Quick demo on row & column level security
* Auto-scale Trino clusters using trino-lb
  * Differences between trino-gateway and trino-lb
```

Reply:
Add that with modifications and a whole bunch of other stuff.